<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CleanRuleSet</class>
 <widget class="QMainWindow" name="CleanRuleSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>432</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>清理规则编辑</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QGroupBox" name="optionName">
        <property name="title">
         <string>清理选项</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QPushButton" name="btnAddOption">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/Window/Images/Add.png</normaloff>:/Window/Images/Add.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="0" rowspan="3">
          <widget class="QListView" name="vSetOption"/>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="btnDeleteOption">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/Window/Images/Remove.png</normaloff>:/Window/Images/Remove.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QGroupBox" name="suffixName">
         <property name="title">
          <string>后缀</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="1">
           <widget class="QPushButton" name="btnAddSuffix">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/Window/Images/Add.png</normaloff>:/Window/Images/Add.png</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="0" rowspan="3">
           <widget class="QListView" name="vSetSuffix"/>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="btnDeleteSuffix">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/Window/Images/Remove.png</normaloff>:/Window/Images/Remove.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QGroupBox" name="fileName">
         <property name="title">
          <string>文件</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="1">
           <widget class="QPushButton" name="btnAddFile">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/Window/Images/Add.png</normaloff>:/Window/Images/Add.png</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="0" rowspan="3">
           <widget class="QListView" name="vSetFile"/>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="btnDeleteFile">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/Window/Images/Remove.png</normaloff>:/Window/Images/Remove.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnSave">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>保存</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/Window/Images/Save.png</normaloff>:/Window/Images/Save.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCancel">
          <property name="text">
           <string>取消</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>CleanRuleSet</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>641</x>
     <y>510</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
